### ISSUE TITLE
Jersey Jitneys

### ISSUE THREAD

**Issue by Sophie-Le-Blanc** [Sophie-Le-Blanc/test#19]
# **Jersey Jitneys (buses) in New Jersey, New Jersey**

**Michael Klatsky
Project Manager – NYC DOT Office of Freight Mobility
Transportation Planning & Management | NYC Department of Transportation 
55 Water St, 6th Floor | New York, NY 10041 | www.nyc.gov/dot
mklatsky@dot.nyc.gov | 212-839-7234**

Notes:

**Comment by Sophie-Le-Blanc** [comment]
June 30 - Currently their lines don't run on schedules, but are working towards making them scheduled. Sent us the GTFS for one line. Waiting on response from Léo-Sam if we can integrate this type of service.

- On hold because not integrating small agencies
